asset/css/markbind.css

```diff
@@ -2,10 +2,19 @@ blockquote {
     font-size: 14px;
 }
 
+.markbind-table {
+    width: auto;
+}
+
 .radio-list-item label {
     font-weight: inherit;
 }
 
+.table-striped > thead,
+.table-striped > tbody > tr:nth-of-type(even) {
+    background-color: #fff;
```

Comment:
Background colour for header and even rows is transparent. When table is on top of a tip-box, the even rows and header will have the same color as the stripes.